Formula/trezor-bridge.rb

```diff
@@ -0,0 +1,26 @@
+class TrezorBridge < Formula
+  desc "Trezor Communication Daemon"
+  homepage "https://github.com/trezor/trezord-go"
+  url "https://github.com/trezor/trezord-go/archive/refs/tags/v2.0.31.tar.gz"
+  sha256 "fd834a5bf04417cc50ed4a418d40de4c257cbc86edca01b07aa01a9cf818e60e"
+  license "LGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build"
+    bin.install "trezord-go"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(output: bin/"trezord-go", ldflags: "-s -w")
```

Reply:
addressed in bb86814f2f57f60fcdf4ab335fe3df560a16ace4